stock_move_ignore_last_purchase_date/views/stock_picking_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <field name="origin" position="after">
+                <field
+                    name="ignore_last_purchase_date"
+                    attrs="{'invisible': [('picking_type_code', '!=', 'incoming')]}"
+                />
+            </field>
```

Comment:
@rinaldifirdaus The field doesn't have to be (shouldn't really be, IMO) in such a noticeable place, although users may not case much about it.  Please check with the client on this point.

Reply:
@AungKoKoLin1997 , client agree to move the position of this field into the other info tab. could you please help to update?thanks